plugins/src/lib.rs

```diff
@@ -708,6 +722,13 @@ where
     }
 }
 
+pub enum FeatureBitsPlace {
+    Node,
+    Channel,
+    Invoice,
+    Init,
+}
```

Comment:
Very nitpicky, but I've seen this twin-enum pattern a number of times, and the simple enumeration of keys in the lightweight one usually ends with `Kind`. Might be worthwhile to adhere to that convention.